mlir/include/mlir/Dialect/Bufferization/Transforms/OneShotAnalysis.h

```diff
@@ -127,9 +127,9 @@ class OneShotAnalysisState : public AnalysisState {
   /// Return true if the buffer of the given tensor value is writable.
   bool isWritable(Value value) const;
 
-  /// Find the definitions of the given tensor value or retrieve them from the
-  /// cache.
-  const SetVector<Value> &findDefinitionsCached(Value value);
+  /// Find the definitions of the given tensor value related to `opOperand` or
```

Comment:
nit: `Find the definitions of the given operand's value or ...`

Reply:
Thanks, fixed.